src/components/widget/List/List.js

```diff
@@ -60,11 +60,11 @@ class ListWidget extends Component {
     if (prevProps.autoFocus !== autoFocus && !isFocused) {
       if (autoFocus) {
         this.handleFocus();
-        !doNotOpenOnFocus && list.size > 1 && this.activate();
+        !doNotOpenOnFocus && list && list.size > 1 && this.activate();
```

Comment:
Is it possible for list to not exist ? Isn't it easier to just always create an empty List object, and have one less point of failure ?

Reply:
I didn't investigate much into this (the reason being that it needs refactoring anyway.)

At some point I got an exception since list was undefined, so I threw that hotfix in there